scripts/qa_crowbarsetup.sh

```diff
@@ -3133,6 +3134,21 @@ function set_proposalvars
     fi
 }
 
+function set_noproxyvar
+{
+    [[ $http_proxy ]] || [[ $https_proxy ]] || return 0
+    [[ $admin_ip ]] || return 0
+    [[ $no_proxy =~ "$adminip" ]] || no_proxy="$adminip,$no_proxy"
+    export no_proxy="${no_proxy%,}";
+    if [[ ! $net_public ]] || [[ $no_proxy =~ $net_public ]] ; then
+        return 0 # only apply this once
+    fi
+    local ips
+    printf -v ips '%s,' $net_public.{1..254}
```

Comment:
I guess this is only for debugging, and should be removed before merging, right?